src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
Q_UNUSED only adds (void*) in front of the line which can also be used to suppress "unchecked return value" warnings.